tooling/operation/get-parameters-as-json-schema.js

```diff
@@ -365,7 +380,7 @@ function getRequestBody(operation, oas) {
     examples.push({ examples: requestBody.examples });
   }
 
-  const cleanedSchema = constructSchema(requestBody.schema, examples);
+  const cleanedSchema = constructSchema(requestBody.schema, examples, undefined, jwtDefaults);
```

Comment:
`currentLocation` should probably always just be an empty string.

```suggestion
  const cleanedSchema = constructSchema(requestBody.schema, examples, '', jwtDefaults);
```